### PR TITLE
ui: bump dependencies

### PR DESCRIPTION
**Pull request by breeswish**
This PR bumps UI dependencies to resolve several UI security issues.

Note: There remains immer issue unresolved due to https://github.com/storybookjs/storybook/issues/16093

### CODE REVIEW

**Review by baurine — APPROVED** [review]

File: ui/config-overrides.js [hunk 1]
```diff
@@ -34,20 +34,6 @@ function isBuildAsDevServer() {
...
 }
 
-const enableEslintIgnore = () => (config) => {
-  const eslintRule = config.module.rules.filter(
-    (r) =>
-      r.use && r.use.some((u) => u.options && u.options.useEslintrc !== void 0)
-  )[0]
-  const options = eslintRule.use[0].options
-  options.ignore = true
-  options.ignorePattern = 'lib/client/api/*.ts'
```
Comment (shhdgit) [inline ui/config-overrides.js:44]: Why can we remote this ignore pattern?

Comment (breeswish) [reply ui/config-overrides.js:44]: There are lint disable hints in the file. For some unknown reasons previously they are not taking effect:

```typescript
/* tslint:disable */
/* eslint-disable */
```

Comment (shhdgit) [reply ui/config-overrides.js:44]: Oops..

**Pull request merged**